bin/dearchive_files.py

```diff
@@ -100,21 +102,35 @@
         files.append(line)
 
 for path in files:
-    abs_path = os.path.abspath(path)
-    fire_path = re.sub(settingsO.get('ftp', 'ftp_mount') + "/", '', abs_path)
-    dearch_f = db.fetch_file(path=abs_path)
-    assert dearch_f is not None, f"File entry with path {abs_path} does not exist in the DB. " \
+    # there is no need for it to take the abspath 
+    #abs_path = os.path.abspath(path)
+    #fire_path = re.sub(settingsO.get('ftp', 'ftp_mount') + "/", '', os.path.abspath(path))
+
+    if re.search("/", path) :
+        #adding a die if the path does not start with nfs
+        if path.startswith("/nfs") is False :
+            sys.exit()
+        dearch_f = db.fetch_file(path=path)
+        assert dearch_f is not None, f"File entry with path {path} does not exist in the DB. " \
```

Comment:
Would it make sense to get this outside the condition, so it checks even for the `else` case?